### ISSUE TITLE
No Enough CodeSignature Space

### ISSUE THREAD

**Issue by Tiny0529** [zhlynn/zsign#65]
0|index  | >>> SignFolder: Frameworks/QiSuDunSdk.framework, (QiSuDunSdk)
0|index  | >>> No Enough CodeSignature Space! Length => Now: 6118, Need: 8666
0|index  | >>> Realloc CodeSignature Space... 
0|index  | 
0|index  | >>> Success!

codesign -vv xx.app
main executable failed strict validation

**Comment by Dershowitz011** [comment]
Did you try to install the app?

**Comment by Tiny0529** [comment]
> Did you try to install the app?

can't install

**Comment by Dershowitz011** [comment]
What was the command you used?

**Comment by Tiny0529** [comment]
> What was the command you used?
./zsign -f -k xx.p12 -m xx.mobileprovision Payload/xx.app
I can send you ipa

**Comment by Dershowitz011** [comment]
Yeah send me. I'll try signing and installing. Send me by uploading to any cloud and share that link here.

**Comment by Dershowitz011** [comment]
@Tiny0529 could you get the app working? If not, send it to me.

**Comment by Tiny0529** [comment]
> @Tiny0529 could you get the app working? If not, send it to me.

https://github.com/Tiny0529/special
testNoEnough.ipa

Thank you
